server/src/lib/model/db/migrations/20230518103600-hide-problematic-latvian-sentences.ts

```diff
@@ -0,0 +1,104 @@
+export const up = async function (db: any): Promise<any> {
+  return db.runSql(`
+    UPDATE sentences SET is_used = FALSE WHERE text in (
```

Comment:
We don't want to overwrite sentences from other locales by accident. So it would be best to add the latvian locale id as constraint as well.

```suggestion
  return db.runSql(`
    UPDATE sentences SET is_used = FALSE WHERE locale_id=(SELECT id from locales WHERE name='lv') AND text in (
```

Reply:
True, better safe than sorry